apps/challenges/views.py

```diff
@@ -3026,16 +3036,28 @@ def scale_resources_by_challenge_pk(request, challenge_pk):
     ):
         response = scale_resources(challenge, worker_cpu_cores, worker_memory)
         if response["ResponseMetadata"]["HTTPStatusCode"] != 200:
-            response_data = {
-                "error": "Issue with ECS."
-            }
+            if response.get('Error', {'Message': 'No error', 'Code': 'No error'}).get('Code', 'No error code') == \
+                    'ClientException':
+                response_data = {
+                    "error": "Challenge workers are inactive or do not exist."
+                }
+            else:
+                response_data = {
+                    "error": "Issue with ECS."
+                }
             return Response(response_data, status=status.HTTP_503_SERVICE_UNAVAILABLE)
         else:
-            response_data = {
-                "Success": "The challenge {} has been scaled successfully".format(
-                    challenge.title
-                )
-            }
+            if response.get("IsWorkerConfigModified", False):
+                response_data = {
+                    "Success": "The challenge \"{}\"'s worker cores and memory were not "
+                               "modified.".format(challenge.title)
+                }
+            else:
+                response_data = {
+                    "Success": "The challenge \"{}\" has been scaled successfully!".format(
+                        challenge.title
+                    )
+                }
```

Comment:
The success message should be `Scaled worker resources successfully!` we don't need to show title of the challenge.